src/Appwrite/Task/CLIPlatform.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Appwrite\Task;
+
+use Utopia\Platform\Platform;
+
+class CLIPlatform extends Platform
```

Comment:
```suggestion
class Tasks extends Platform
```

Reply:
I also need a tasks service in the same namespace, struggling with good name, I named it `TasksService` and the platform `Tasks`. WDYT?